wmn-data.json

```diff
@@ -2323,8 +2263,8 @@
         "e_code" : 200,
         "e_string" : " | HackerRank</title>",
         "m_string" : ":: HackerRank</title>",
-        "m_code" : 302,

```

Comment:
I'm seeing that we get a 302 respoinise code first when an incorrect username is submitted and the it redirects to a 101 then 200 response code pages. Can you check on this?